src/network/networkd-setlink.c

```diff
@@ -958,7 +993,9 @@ static int link_up_or_down_handler_internal(sd_netlink *rtnl, sd_netlink_message
                 goto on_error;
 
         r = sd_netlink_message_get_errno(m);
-        if (r < 0) {
+        if (r == -ENETDOWN && up && link_up_dsa_slave(link) > 0)
```

Comment:
why is it printing the error when `link_up_dsa_slave` returns success?

Reply:
This intends to log the first attempt is failed in log level. The log message below is slightly updated.